life_expectancy/tests/test_cleaning.py

```diff
@@ -1,16 +1,37 @@
 """Tests for the cleaning module"""
 import pandas as pd
-import life_expectancy.cleaning as cleaning
+from unittest import mock
+import life_expectancy.main as main
+from life_expectancy.data_loading import load_data
+from life_expectancy.data_loading import save_data
+from life_expectancy.data_cleaning import clean_data
 
 from . import OUTPUT_DIR
 
+def test_main(pt_life_expectancy_expected):
 
-def test_clean_data(pt_life_expectancy_expected):
     """Run the main function and compare the output to the expected output"""
-    cleaning.main('PT')
-    pt_life_expectancy_actual = pd.read_csv(
-        OUTPUT_DIR / "pt_life_expectancy.csv"
-    )
-    pd.testing.assert_frame_equal(
-        pt_life_expectancy_actual, pt_life_expectancy_expected
-    )
+    dataframe = main.main("PT")
```

Comment:
<mandatory> there's no test for main function

Reply:
I think there was, however, I have updated the code and created a separate testing module for the main function